docs/use-the-network/run-a-network-server/run-console.mdx

```diff
@@ -139,15 +143,28 @@ Copy the local docker-compose file
 cp templates/docker-compose-local.yaml docker-compose.yaml
 ```
 
-`docker-compose up` builds Console and launches it with two other containers:
-Router and Postgres.
+This template builds console locally and does not use nginx for SSL. You may
+want to read about nginx setup [here](/use-the-network/run-a-network-server/deploy-console).
+
+If you launch docker-compose from here, you will build from the `dev` branch.
```

Comment:
master branch?

Reply:
didn't we just realize that when i did git clone and no branch chases that i am in fact building from `dev`?